### PR TITLE
Removed underscore prefix from property names in Controller files

### LINKED ISSUE #16278: Remove underscore prefix from property names
From [PSR-12](https://www.php-fig.org/psr/psr-12/#43-properties-and-constants):
> Property names MUST NOT be prefixed with a single underscore to indicate protected or private visibility. That is, an underscore prefix explicitly has no meaning.

Example:
```diff
-     private $_user;
+     private $user;
```

Files detected by phpcs: `vendor/bin/phpcs --warning-severity=4`

FILE | WARNINGS
-----|---------
libraries/classes/Bookmark.php | 5
libraries/classes/Console.php | 2
libraries/classes/DatabaseInterface.php | 9
libraries/classes/Encoding.php | 4
libraries/classes/File.php | 10
libraries/classes/Footer.php | 4
libraries/classes/Header.php | 12
libraries/classes/Index.php | 13
libraries/classes/IndexColumn.php | 6
libraries/classes/LanguageManager.php | 6
libraries/classes/Menu.php | 2
libraries/classes/OutputBuffering.php | 4
libraries/classes/RecentFavoriteTable.php | 3
libraries/classes/Response.php | 8
libraries/classes/SavedSearches.php | 6
libraries/classes/Scripts.php | 2
libraries/classes/Table.php | 3
libraries/classes/Theme.php | 1
libraries/classes/ThemeManager.php | 2
libraries/classes/Tracker.php | 1
libraries/classes/TwoFactor.php | 3
libraries/classes/Types.php | 1
libraries/classes/Config/ConfigFile.php | 9
libraries/classes/Config/Form.php | 2
libraries/classes/Config/FormDisplay.php | 9
libraries/classes/Config/PageSettings.php | 4
libraries/classes/Config/Forms/BaseFormList.php | 1
libraries/classes/Controllers/Table/FindReplaceController.php | 3
libraries/classes/Controllers/Table/SearchController.php | 7
libraries/classes/Controllers/Table/StructureController.php | 1
libraries/classes/Controllers/Table/ZoomSearchController.php | 7
libraries/classes/Database/Qbe.php | 27
libraries/classes/Gis/GisGeometryCollection.php | 1
libraries/classes/Gis/GisLineString.php | 1
libraries/classes/Gis/GisMultiLineString.php | 1
libraries/classes/Gis/GisMultiPoint.php | 1
libraries/classes/Gis/GisMultiPolygon.php | 1
libraries/classes/Gis/GisPoint.php | 1
libraries/classes/Gis/GisPolygon.php | 1
libraries/classes/Gis/GisVisualization.php | 4
libraries/classes/Plugins/TwoFactorPlugin.php | 3
libraries/classes/Plugins/Auth/AuthenticationCookie.php | 2
libraries/classes/Plugins/Export/ExportCodegen.php | 1
libraries/classes/Plugins/Export/ExportPdf.php | 2
libraries/classes/Plugins/Export/ExportSql.php | 1
libraries/classes/Plugins/Export/ExportXml.php | 2
libraries/classes/Plugins/Import/ImportCsv.php | 1
libraries/classes/Plugins/Import/ImportMediawiki.php | 1
libraries/classes/Plugins/Schema/Dia/DiaRelationSchema.php | 6
libraries/classes/Plugins/Schema/Eps/EpsRelationSchema.php | 3
libraries/classes/Plugins/Schema/Pdf/Pdf.php | 7
libraries/classes/Plugins/Schema/Pdf/PdfRelationSchema.php | 15
libraries/classes/Plugins/Schema/Pdf/TableStatsPdf.php | 1
libraries/classes/Plugins/Schema/Svg/SvgRelationSchema.php | 7
libraries/classes/Plugins/TwoFactor/Application.php | 1
libraries/classes/Properties/Options/OptionsPropertyGroup.php | 1
libraries/classes/Properties/Options/OptionsPropertyItem.php | 3
libraries/classes/Properties/Options/OptionsPropertyOneItem.php | 5
libraries/classes/Properties/Options/Groups/OptionsPropertySubgroup.php | 1
libraries/classes/Properties/Plugins/ExportPluginProperties.php | 1
libraries/classes/Properties/Plugins/PluginPropertyItem.php | 5
libraries/classes/Server/SysInfo/WindowsNt.php | 1
test/classes/DatabaseInterfaceTest.php | 1
test/classes/IndexTest.php | 1
test/classes/VersionInformationTest.php | 1
test/classes/Controllers/Table/RelationControllerTest.php | 1
test/classes/Controllers/Table/SearchControllerTest.php | 1
test/classes/Controllers/Table/StructureControllerTest.php | 1
test/classes/Stubs/DbiDummy.php | 1
test/classes/Stubs/Response.php | 2
test/selenium/CreateRemoveUserTest.php | 2

### PR DESCRIPTION
**Pull request by roshnaeem**
### Description

I have removed underscore prefixes from property names in all the files present in Controller directories. 

Fixes #16278 

Signed-off-by: Rosheen Naeem <rosheennaeem4@gmail.com>

### CODE REVIEW

File: test/classes/Controllers/Table/RelationControllerTest.php [hunk 1]
```diff
@@ -84,7 +84,8 @@ public function exists($name): bool
 
         $GLOBALS['dbi'] = $dbi;
 
-        $this->_response = new ResponseStub();
+        $this->
+        response = new ResponseStub();
```
Comment (williamdes) [inline test/classes/Controllers/Table/RelationControllerTest.php:88]:
```suggestion
        $this->response = new ResponseStub();
```

**Comment by williamdes** [comment]
Good contribution !
Could you fix https://travis-ci.org/github/phpmyadmin/phpmyadmin/jobs/715424698#L443 and the comment I posted please ?
Also feel free to squash your commits in one

One more note, your GIT could be misconfigured `From: Rosheen Naeem <rosheen@xgrid.co>` https://github.com/phpmyadmin/phpmyadmin/commit/a51c96bdb3dda49c961dcce77a28b5d5b396fb92.patch

**Comment by roshnaeem** [comment]
@williamdes I tried removing sign off rosheen@xgrid.co but I could not. 
I have amended my commit a lot of time using "git commit --amend" to remove it from Notepad file of the message but every time when I save the commit message and open it again it is still there. Can you help me with this matter?

**Comment by williamdes** [comment]
@roshnaeem I think you should read https://www.git-tower.com/learn/git/faq/change-author-name-email and execute the 2 commands to change your name and email 
afterwards `git commit --amend --reset-author` will help you but you will maybe need to do a `git rebase --interactive` to squash the commits before

**Comment by roshnaeem** [comment]
@williamdes  can you check this commit, please. I have tried to solve the issues. 
https://github.com/phpmyadmin/phpmyadmin/pull/16292/commits/3d1a5a8fdcd26f2774ced1883fd09b474007746f

**Comment by williamdes** [comment]
@roshnaeem I think you did not use `--force` on the `git push` command.
I would recommend you to `git reset --soft 22c4c2e1e02497d0e280b4360cd49f296e3da873` See: https://github.com/roshnaeem/phpmyadmin/commits/master

Then commit and `git push --force`

Here is a GitKraken view of the history
image

**Comment by roshnaeem** [comment]
I have made a new pull request. I am sorry I am new to Github. It might also have some issues.
https://github.com/phpmyadmin/phpmyadmin/pull/16297

**Pull request closed without merging**